pkg/conversion/converter.go

```diff
@@ -495,15 +495,6 @@ func (c *Converter) callCustom(sv, dv, custom reflect.Value, scope *scope) error
 // one is registered.
 func (c *Converter) convert(sv, dv reflect.Value, scope *scope) error {
 	dt, st := dv.Type(), sv.Type()
-	// Apply default values.
-	if fv, ok := c.defaultingFuncs[st]; ok {
```

Comment:
`defaultConvert` may not have default values populated from the look of the code: https://github.com/kubernetes/kubernetes/blob/master/pkg/conversion/converter.go#L521

Does it matter though?